src/ui/public/time_buckets/calc_es_interval.js

```diff
@@ -45,3 +46,6 @@ export default function () {
 
   return esDuration;
 }
+
+// used by x-pack. TODO: switch to named and remove.
```

Comment:
I'd suggest removing this comment and others like it that reference x-pack from OSS Kibana source code. When we see these default imports in x-pack-kibana, we'll know which ones to come back and cleanup in OSS Kibana.